src/Dfe.PlanTech.Core/DataTransferObjects/Sql/SqlRecommendationDto.cs

```diff
@@ -3,10 +3,9 @@
 public class SqlRecommendationDto : ISqlDto
 {
     public int Id { get; init; }
-    public string? RecommendationText { get; init; } = null!;
     public string ContentfulSysId { get; init; } = null!;
+    public string? RecommendationText { get; init; } = null!;
     public DateTime DateCreated { get; init; } = DateTime.UtcNow;
-    public int QuestionId { get; init; }
-    public SqlQuestionDto Question { get; init; } = null!;
+    public string QuestionContentfulRef { get; init; } = null!;
     public bool Archived { get; init; } = false;
```

Comment:
Discussion on call - review if moving away from ID to Contentful Ref is desirable.

If it's for the audit history entries etc. (in the call something like "status updated to x because question y had answer z" was mentioned), presumably we need/want the question and answer as it existed at the time of submitting the self assessment.